tools/generate_fa_sysdata.R

```diff
@@ -98,6 +98,4 @@ for (i in seq_along(fa_list)) {
 }
 
 # Create `sysdata.rda`
-if (fontawesome:::audit_fa_tbl(tbl = fa_tbl)) {
-  usethis::use_data(fa_tbl, internal = TRUE, overwrite = TRUE)
-}
+usethis::use_data(fa_tbl, internal = TRUE, overwrite = TRUE)
```

Comment:
Does it make sense to move the `audit_fa_tbl` function into this script and still use it?

Reply:
Yes, makes a lot of sense. We can have all the testing that was there before but put it in the dev scripts.